src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -57,46 +62,78 @@
 from sentry.utils.cursors import Cursor, CursorResult
 from sentry.utils.dates import get_rollup_from_request, parse_stats_period
 
-# These are the use case ids that are queried by default in case no use case is supplied.
-DEFAULT_USE_CASE_IDS = [
-    UseCaseID.TRANSACTIONS,
-    UseCaseID.SESSIONS,
-    UseCaseID.SPANS,
-    UseCaseID.CUSTOM,
-    UseCaseID.PROFILES,
-    UseCaseID.METRIC_STATS,
-]
 
+def can_access_use_case_id(request: Request, use_case_id: UseCaseID) -> bool:
+    api_access = get_use_case_id_api_access(use_case_id)
+    return api_access == UseCaseIDAPIAccess.PUBLIC or (
+        has_elevated_mode(request) and api_access == UseCaseIDAPIAccess.PRIVATE
```

Comment:
looking at the comments in `has_elevated_mode` we probably should not rely to that, i think that `is_active_superuser` is what we want here

Reply:
I was told by @markstory that this would be the correct way to check it and it will be refactored in the future to a newer function.

Reply:
If you check the definition of the function it uses `is_active_superuser`. https://github.com/getsentry/sentry/blob/9e8670622e9b2c04dc1eab32b41886db19668d55/src/sentry/auth/elevated_mode.py#L68-L68